### DISCUSSION TITLE
Timezone map in set streak hour offset

### DISCUSSION THREAD

**Discussion by Whydidthechickencrosstheroad** [monkeytypegame/monkeytype#5503] [Ideas] [answered]
So, some people get confused without a map. Like this: If you set the time 1 hour away from the UTC (Coordinated Universal Time) but you live in PST, you don't know where that is. If it is selected which timezone you chose like UTC-8, it will show the map in color. This is the timezone map but if you select a timezone you chose, it will be highlighted in a color (not the colors in the map). The timezone map picture: 
![World_Time_Zones_Map svg](https://github.com/monkeytypegame/monkeytype/assets/158355086/80c4e46e-79e5-4ce3-b5a3-eeaa01e4e519)

**Comment by Whydidthechickencrosstheroad** [comment]
Why is this not added yet? My friend is confused without timezone map

**Reply by fehmer** [reply, marked as answer]
I think the current interface is pretty clear. It shows the new reset time in your local timezone.
![image](https://github.com/user-attachments/assets/ad5719e6-9e12-45d1-a7c3-d0a5aa3aa0b3)
